core/controllers/editor_test.py

```diff
@@ -697,6 +702,73 @@ def test_deletion_rights_for_published_exploration(self):
         self.assertEqual(response.status_int, 200)
         self.logout()
 
+    def test_logging_info_after_deletion(self):
+        """Tests correctness of logged statements while deleting exploration"""
+        logging_info = []
+
+        def add_logging_info(msg, *_):
+            if msg != 'all_pending: clear %s':
+                logging_info.append(msg)
+
+        with self.swap(logging, 'info', add_logging_info), self.swap(
+            logging, 'debug', add_logging_info):
+            # checking for normal user
+            exp_id = 'unpublished_eid'
+            exploration = exp_domain.Exploration.create_default_exploration(
+                exp_id)
+            exp_services.save_new_exploration(self.owner_id, exploration)
+
+            self.login(self.OWNER_EMAIL)
+            self.testapp.delete(
+                '/createhandler/data/%s' % exp_id, expect_errors=True)
+            self.assertEqual(logging_info[0],
+                             self.owner_id + ' tried to delete exploration ' +
```

Comment:
(a) OK to hardcode the expected error.

(b) In this codebase, use interpolation instead of concatenation for strings. I.e. '%s tried to .. %s' % (..., ...)